docs/connectivity/Agent.mdx

```diff
@@ -0,0 +1,275 @@
+---
+id: Agent
+title: Instrument Agents
+displayed_sidebar: webUiSidebar
+---
+
+import { PlusCircleOutlined, 
+         SelectOutlined,
+         DiffOutlined
+       } from '@ant-design/icons';
+
+The Ganymede Agent is an executable that can be installed on lab instruments running Windows or Linux to capture instrument data systematically.  The agent can
+
+- filter and designate how to process data
+- interact with connected instruments
+- upload files into Ganymede Cloud
+- initiate data processing pipelines
+
+By installing Agents, your lab machines become an integrated part of the Ganymede system, enhancing the data management and processing capabilities in your lab.
+
+## What does an Agent consist of?
```

Comment:
This section definitely excludes capabilities of agent to listen on a cron interval or for cloud file changes.